### PR TITLE
couchdb: Update 2.0.0

### PR DESCRIPTION
**Pull request by arete**
Signed-off-by: Gerardo Di Iorio arete74@gmail.com

### CODE REVIEW

File: srcpkgs/couchdb/files/couchdb/run [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 export HOME=/var/lib/couchdb
+export ERL_FLAGS=-couch_ini /usr/lib/couchdb/etc/default.ini /usr/lib/couchdb/etc/datadirs.ini /etc/couchdb/local.ini"
```
Comment (jannisch) [inline srcpkgs/couchdb/files/couchdb/run]: there is no quotation mark that initiates the string

**Comment by jannisch** [comment]
This probably does not work 100% yet. xbps-install was not able to run the `chmod` after installation.

**Comment by ebfe** [comment]
Did anyone try this?

**Comment by arete** [comment]
I have update bad branch.
i now fix this

**Comment by arete** [comment]
hi @ebfe, i have update the corrrect branch.

**Comment by revington** [comment]
Can this get merged?

File: srcpkgs/couchdb/template [hunk 2]
```diff
@@ -1,38 +1,52 @@
...
-fi
 
-pre_configure() {
-	sed -i 's|-Werror||g' src/couchdb/priv/Makefile.*
+
 if [ "$CROSS_BUILD" ]; then
-	sed -i '/FLAGS.*\(usr\|opt\)/s/.*/:/' configure
+	makedepends+=" erlang"
 fi
+do_install() {
```
Comment (the-maldridge) [inline srcpkgs/couchdb/template]: Why does updating to a new version require overriding the entire do_install() function.  Does couchdb just not install with the standard one?

Comment (arete) [inline srcpkgs/couchdb/template]: Not, couchdb not have standard install.
Is useful have couchdb1 and couchdb2,0 in the repo ?

Comment (jannisch) [inline srcpkgs/couchdb/template]: I don't think so. CouchDB 2 is compatible with CouchDB 1.6 databases and API.

**Comment by arete** [comment]
ok, i rebase pull request  and try to fix cross build.

2017-06-24 19:59 GMT+02:00 Jannis Christ <notifications@github.com>:

**Comment by arete** [comment]
hi, i try to build couchdb2.0 with erlang20, but not work https://github.com/apache/couchdb/issues/617 .
Wait for upstream patch or new release.

**Comment by revington** [comment]
@arete looks like https://github.com/apache/couchdb/issues/617 has been solved.

**Comment by arete** [comment]
Hi, thanks for info.
I wait couchdb 2.1.1, now in rc1, forma full support of erlang 20.
I have build the rc1 and work fine...now i try to fix cross compile andò
wait forma final release.


Il 01 nov 2017 20:15, "Pedro Narciso García Revington" <
notifications@github.com> ha scritto:

**Comment by arete** [comment]
Hi i have branch for couchdb 1.7.0 and couchdb 2.1.1
for now in rc.

https://github.com/arete/void-packages/tree/couchdb2.1.1 couchdb2.1.1
https://github.com/arete/void-packages/tree/couchdb1.7.0  couchdb1.7.0

wait for final relase for pull request.

**Comment by jannisch** [comment]
@arete it's out now in case you want to update and submit the PR

**Pull request closed without merging**